### PR TITLE
Ep 12391 listview common component and outside click scope feature

### PR DESCRIPTION
**Pull request by JerryDoubleU**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,16 @@ This library has two packages:
...
+    2. `ep-asset-list` - asset list
+    3. `ep-colorpicker` - a colorpicker
+    4. `ep-dialog` - a dialog that can display arbitrary content.
+    5. `ep-displays-counter` - displays a "monitor" icon with a number inside.
+    6. `ep-dropdown-select` - a dropdown select with customizable list elements
+    7. `ep-select-button` - a collection of buttons side-by-side. Only one can be selected at once.
+    8. `ep-sort-button` - a button useful for sorting tables
+    9. `ep-tag` - a rectangle with an 'x' on the side.
+    10. `ep-theme-picker` - theme picker.
+    11. `ep-theme-thumbnail` - theme thumbnail, used by theme picker.
```
Comment (sulco) [inline README.md]: This list would probably be more readable as a table. Just a thought.

File: dashboards/asset-list-table/asset-list-table.component.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+@Component({
+  selector: 'ep-asset-list-table',
+  templateUrl: './asset-list-table.component.html',
+  styleUrls: ['./asset-list-table.component.scss']
+})
+export class AssetListTableComponent {
+  @Input() assets: Asset<AssetValueType>[];
+  @Output() onOpen = new EventEmitter<Asset<AssetValueType>>();
```
Comment (sulco) [inline dashboards/asset-list-table/asset-list-table.component.ts]: There are 17 `Asset<AssetValueType>` here, often themselves in generics. In such cases I'd consider making a type alias, e.g. `type AssetValue = Asset<AssetValueType>` - should increase readability.

File: dashboards/click-outside/click-outside.directive.spec.ts [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        </div>`
+})
+class TestScopeClickOutsideComponent {
+    onClickOutside() {}
+}
+
+describe('ClickOutsideDirective', () => {
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [
```
Comment (sulco) [inline dashboards/click-outside/click-outside.directive.spec.ts]: indentation went to space here as well :)

File: README.md (hunk 1 above)
Comment (JerryDoubleU) [reply README.md]: That's a good idea to improve the components package docs. I'll write that down and have a closer look later.

**Pull request merged**